src/Components/BookInfo/BookInfo.css

```diff
@@ -13,24 +15,72 @@
 
 .book-image {
   grid-area: 1 / 1 / 4 / 2;
+  margin: 20px;
 }
 
 .title-and-author {
+  border-radius: 10px;
   grid-area: 1 / 2 / 2 / 4;
 }
 
 .description {
+  border-radius: 10px;
+  display: flex;
   grid-area: 2 / 2 / 3 / 4;
 }
 
+.desciption-text {
+  margin:auto;
+}
+
 .button-container {
   grid-area: 3 / 2 / 4 / 4;
   display: flex;
   margin: auto
 }
 
-button {
+.add-to-reading-list, .purchase {
+  background-color: whitesmoke;
+  border-radius: 5px;
+  color: teal;
+  font-family: 'Amatic SC', cursive;
+  font-size: 24px;
+  font-weight: 500;
   height: 50px;
   margin: 20px;
+  text-decoration: none;
   width: 100px;
 }
+
+.add-to-reading-list:hover, .purchase:hover {
+  background:  #F4DBD8;
+  transition: 0.5s;
+}
+
+@media only screen and (max-width: 800px) {
```

Comment:
This styling looks really great!! And great thought to start making our website responsive!!

Reply:
Thanks! I can do similar to the other components if it seems like a breakpoint we want to go with.